DSA-Campaign-Uplift-Estimation/src/test/java/com/google/sps/ServletsTests.java

```diff
@@ -20,7 +20,7 @@
 import org.junit.runners.JUnit4;
 
 @RunWith(JUnit4.class)
-public final class EmptyTest {
+public final class ServletsTests {
```

Comment:
Is this file for combining all the servlet test? If so might be better to have a testing file for each file you want to test.